rake/build/environment.rb

```diff
@@ -79,8 +79,8 @@
 pipeopts 'st2mistral' do
   checkout true
   envpass :giturl,  'https://github.com/StackStorm/mistral', from: 'ST2MISTRAL_GITURL'
-  envpass :gitrev,  'st2-1.3.2',                             from: 'ST2MISTRAL_GITREV'
+  envpass :gitrev,  'master',                                from: 'ST2MISTRAL_GITREV'
```

Comment:
Here you changed `default` values, which works if you don't pass env variables like `ST2MISTRAL_GITREV`.

You need to change this part: https://github.com/StackStorm/st2-packages/blob/85fffb4e2ad16561bd5d5c50ddb7b0e2fc50d1eb/.circle/buildenv.sh#L59-L69